### PR TITLE
Mukuro - My Room

### PR DESCRIPTION
**Pull request by contingency40**
All of Mukuro's My Room dialogue should be complete except for the engagement scenes.

There may be a conflict once the 2-2 PR is merged as some scenes reused dialogue from there.

### CODE REVIEW

File: Gaijinizer patch/Map080.txt [hunk 1]
```diff
@@ -135,38 +140,40 @@ Well, let's play ...#MTLed
...
 私を奈落の闇から救ってくれたこと……
 
 そして、兄様の『想い』を救ってくれたことに対して……
 > CONTEXT: Map080/events/5/pages/1/35/Dialogue/enemy4
-That saved me from the darkness of abyss ......
-And for saving my brother's "thought" ...#MTLed
+For saving me from the darkness of the abyss...
+
+And for saving my brother's "thoughts"...#Ron
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/Map080.txt]: Note

File: Gaijinizer patch/Map080.txt [hunk 2]
```diff
@@ -175,77 +182,83 @@ And for saving my brother's "thought" ...#MTLed
...
+soul so we could die together. But being able to
+use it purely for your pleasure isn't bad either.#Ron
 > END STRING
 
 > BEGIN STRING
 でも、搾精は強引なままと……
 > CONTEXT: Map080/events/5/pages/1/45/Dialogue/kazuya
-But, milking is forced to remain ...#MTLed
+But, forced milking is still...#RonIffy
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/Map080.txt]: Note

File: Gaijinizer patch/Map080.txt [hunk 3]
```diff
@@ -175,77 +182,83 @@ And for saving my brother's "thought" ...#MTLed
...
 私は、普通に礼をしたいわけでも、ただの搾精人形でもない。
 
 そこを勘違いしないようにな。
 > CONTEXT: Map080/events/5/pages/1/55/Dialogue/enemy4
-I'm not a normal thank-you, not just a milking
-doll. Don't get me wrong.#MTLed
+Don't get the wrong idea. I'm not just some
+ordinary milking doll, and my way of thanking
+you won't be what you'd expect.#RonIffy
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/Map080.txt]: Note

File: Gaijinizer patch/Map080.txt [hunk 4]
```diff
@@ -352,77 +363,81 @@ just that.#MTLed
...
 \\N[1]を犯すという目標がなければ、
 
 きっとあの空間を動くことすら嫌だったろう。
 > CONTEXT: Map080/events/5/pages/1/127/Dialogue/enemy4
-Without the goal of committing \\N[1],
-Surely I would not like to move that space.#MTLed
+If I hadn't had the goal of raping you, \\N[1],
+I'm sure I wouldn't have had the will to move out
+of that space.#RonIffy
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/Map080.txt]: Note

File: Gaijinizer patch/Map080.txt [hunk 5]
```diff
@@ -352,77 +363,81 @@ just that.#MTLed
...
 
 もともと、私には無用の産物だったのさ。
 > CONTEXT: Map080/events/5/pages/1/131/Dialogue/enemy6
-Well...now I'm released from twins and I don't
-have that ability. Originally, it was a useless
-product to me.#MTLed
+Well... I don't have that ability now that I'm
+free of the Twins. It was always a useless
+ability for me anyway.#Ron
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/Map080.txt]: Note

File: Gaijinizer patch/Map080.txt [hunk 6]
```diff
@@ -431,94 +446,101 @@ I could not attack me ...#MTLed
...
 物言いも謎だったしね……
 
 『我』がどうとか……
 > CONTEXT: Map080/events/5/pages/1/171/Dialogue/kazuya
-The story was a mystery too ...
-"I" is how .........#MTLed
+The way they spoke was mysterious too...
+
+Like why they always emphasized the word "I"...?#Ron
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/Map080.txt]: Note

File: Gaijinizer patch/Map080.txt [hunk 7]
```diff
@@ -565,76 +591,84 @@ Is a doctor! ? Why are you here?#MTLed
...
 
 躯とやら！！
 > CONTEXT: Map080/events/5/pages/1/281/Dialogue/other3
-Your physical and motion comfort is completely
-different!!Mukuro Toya! !#MTLed
+Your body's movements should feel much better
+now!!
+
+Mukuro Person!!#RonIffy
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/Map080.txt]: Note

File: Gaijinizer patch/Troops.txt [hunk 8]
```diff
@@ -6122,40 +6132,44 @@ Mukuro 【Whole book】 missionary position#MTLed
...
+
+I'll make you ejaculate over and over...#Ron
 > END STRING
 
 > BEGIN STRING
 \\N[1]は躯に後背位で強引に犯されてしまった！！
 > CONTEXT: Troops/126/pages/0/40/Dialogue
-\\N[1] has been forced by Mukuro in doggy style! !#MTLed
+\\N[1] is being forcibly raped by Mukuro in doggy style!!#Ron
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/Troops.txt]: Note

File: Gaijinizer patch/Troops.txt [hunk 9]
```diff
@@ -6122,40 +6132,44 @@ Mukuro 【Whole book】 missionary position#MTLed
...
 > BEGIN STRING
 \\N[1]は、髪で拘束されてそのまま強引に歩かされている…
 
 その先には、こちらにお尻を向けた躯が待ち構えている…
 > CONTEXT: Troops/126/pages/0/15/Dialogue
-\\N[1] is bound with hair and walks as it is.
-Beyond that, Mukuro with her butt is waiting for you ...#MTLed
+\\N[1] is bound with Mukuro's hair and forced to walk over
+to her. Waiting for him is Mukuro with her rear pointed
+in his direction...#Ron
```
Comment (Triketos) [inline Gaijinizer patch/Troops.txt]: Flow and a bit of rearranging...:
>_**\\\N[1] is bound with hair and forced to walk over
as Mukuro waits with her rear pointed at him...**_

File: Gaijinizer patch/Map080.txt (hunk 6 above)
Comment (contingency40) [reply Gaijinizer patch/Map080.txt]: I just had a check and it looks like I translated all the ones I found from "I" to "me" for grammatical reasons.

Comment (Triketos) [reply Gaijinizer patch/Map080.txt]: Probably best to swap "I" with "Me" here, then.

**Pull request merged**